src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -151,6 +157,7 @@ private IServiceProvider ConfigureIOC(IServiceCollection services)
                 option.UseSqlServer(Configuration.SqlConnectionString, options => options.EnableRetryOnFailure(3));
 
                 config.For<AssessorDbContext>().Use(c => new AssessorDbContext(option.Options));
+                config.For<IDbConnection>().Use(c => new SqlConnection(Configuration.SqlConnectionString));
```

Comment:
Looks like this is being added twice.  Merge issue?

Reply:
Yep will take a look - probably merge let's see, is easy enough to fix.

Reply:
done